pyproject.toml

```diff
@@ -96,3 +96,6 @@ line-length = 100  # Only for comments, as code is handled by black at length 88
 
 [tool.ruff.pydocstyle]
 convention = "google"
+
+[tool.bandit]
+skips = ["B101", "B108", "B307", "B404", "B601", "B602"]
```

Comment:
Can we fix or locally ignore some of these issues instead of skipping them entirely?

We should be able to skip B101 for non-test files only. See https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html